catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -216,4 +227,32 @@ public void setEnforceWarnings(boolean enforceWarnings) {
     public boolean getEnforceWarnings() {
         return enforceWarnings;
     }
+
+    protected class ValidationResult {
+        boolean errors;
```

Comment:
If you keep these package private you don't need the getters/setters. Otherwise id say make these private.